frontend/packages/kubevirt-plugin/src/components/vms/vmi-details-page.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import { navFactory, HintBlock, ExternalLink } from '@console/internal/components/utils';
+import { DetailsPage } from '@console/internal/components/factory';
+import { PodModel } from '@console/internal/models';
+import { VMDisksFirehose } from '../vm-disks';
+import { VMNics } from '../vm-nics';
+import {
+  VirtualMachineInstanceMigrationModel,
+  VirtualMachineInstanceModel,
+  VirtualMachineModel,
+} from '../../models';
+import { getResource } from '../../utils';
+import {
+  VM_DETAIL_OVERVIEW_HREF,
+  VM_DETAIL_DISKS_HREF,
+  VM_DETAIL_NETWORKS_HREF,
+  VM_DETAIL_CONSOLES_HREF,
+} from '../../constants';
+import { VMEvents } from './vm-events';
+import { VMConsoleFirehose } from './vm-console';
+import { VMDetailsFirehose } from './vm-details';
+import { menuActionsCreator } from './menu-actions';
+import { VMDashboard } from './vm-dashboard';
+
+import './vmi-details-page.scss';
+
+export const VirtualMachinesInstanseDetailsPage: React.FC<VirtualMachinesDetailsPageProps> = (
+  props,
+) => {
+  const { name, ns: namespace } = props.match.params;
+
+  const breadcrumbsForVMPage = (match: any) => () => [
+    {
+      name: VirtualMachineModel.labelPlural,
+      path: `/k8s/ns/${match.params.ns || 'default'}/virtualmachines`,
+    },
+    { name: `${match.params.name} Details`, path: `${match.url}` },
+  ];
+
+  const dashboardPage = {
+    href: '', // default landing page
+    name: 'Dashboard',
+    component: VMDashboard,
+  };
+
+  const overviewPage = {
+    href: VM_DETAIL_OVERVIEW_HREF,
+    name: 'Overview',
+    component: VMDetailsFirehose,
+  };
+
+  const consolePage = {
+    href: VM_DETAIL_CONSOLES_HREF,
+    name: 'Consoles',
+    component: VMConsoleFirehose,
+  };
+
+  const nicsPage = {
+    href: VM_DETAIL_NETWORKS_HREF,
+    name: 'Network Interfaces',
+    component: VMNics,
+  };
+
+  const disksPage = {
+    href: VM_DETAIL_DISKS_HREF,
+    name: 'Disks',
+    component: VMDisksFirehose,
+  };
+
+  const pages = [
+    dashboardPage,
+    overviewPage,
+    navFactory.editYaml(),
+    consolePage,
+    navFactory.events(VMEvents),
+    nicsPage,
+    disksPage,
+  ];
+
+  const resources = [
+    getResource(PodModel, { namespace, prop: 'pods' }),
+    getResource(VirtualMachineInstanceMigrationModel, { namespace, prop: 'migrations' }),
+  ];
+
+  return (
+    <DetailsPage
+      {...props}
+      name={name}
+      namespace={namespace}
+      kind={VirtualMachineInstanceModel.kind}
+      kindObj={VirtualMachineInstanceModel}
+      menuActions={menuActionsCreator}
+      pages={pages}
+      resources={resources}
+      breadcrumbsFor={breadcrumbsForVMPage(props.match)}
+    >
+      <HintBlock
```

Comment:
I changes the text to make sense also when this VMI is owned :-)

p.s.
for this PR users should not end up in a VMI details if it's owned because we removed the easy links to it.

Reply:
I don't know if this is a good solution. Please consider removing it or making it conditional. 

Btw they will end up here quite easily when they click on a link in VM events